src/PresentationalComponents/Notification/Notification.js

```diff
@@ -109,20 +113,25 @@ const Notification = () => {
   return (
     <React.Fragment>
       <PageHeader>
-        <PageHeaderTitle title="Notification Insights" />
+        <Split>
+          <SplitItem isFilled>
+            <PageHeaderTitle
+              title={`My Notifications | ${bundleDisplayTitle.title}`}
```

Comment:
don't forget to update this one too
```suggestion
              title={`My Notifications | ${bundleDisplayTitle}`}
```